srcpkgs/ferium/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'ferium'
+pkgname=ferium
+version=4.3.4
+revision=1
+build_style=cargo
+build_helper=qemu
+short_desc="CLI program for managing Minecraft mods and modpacks"
+maintainer="yosh <yosh-git@riseup.net>"
+license="MPL-2.0"
+homepage="https://github.com/gorilla-devs/ferium"
+changelog="https://raw.githubusercontent.com/gorilla-devs/ferium/main/CHANGELOG.md"
+distfiles="https://github.com/gorilla-devs/ferium/archive/refs/tags/v${version}.tar.gz"
+checksum=7f5fab2141a03c6fcf22dac2349331f2a478a48677d91defc545406ce70f7aca
+
+post_build() {
+	FERIUM="target/${RUST_TARGET}/release/ferium"
+	vtargetrun ${FERIUM} complete bash > ferium.bash
+	vtargetrun ${FERIUM} complete zsh > ferium.zsh
+	vtargetrun ${FERIUM} complete fish > ferium.fish
+}
+
+post_install() {
+	vcompletion ferium.bash bash
+	vcompletion ferium.zsh zsh
+	vcompletion ferium.fish fish
+}
```

Comment:
See e.g. https://github.com/void-linux/void-packages/blob/6795b29716be848becdfc53f176f85820c29f7de/srcpkgs/fd/template for an example how it's preferred to be done

Reply:
ah, I guess I looked at some un-updated templates x)